installer/datafiles/base_omsagent.data

```diff
@@ -208,39 +208,52 @@ fi
 #
 # Thus, if we're an upgrade, skip all of this cleanup
 ETC_DIR=/etc/opt/microsoft/omsagent
+VAR_DIR=/var/opt/microsoft/omsagent
 WORKSPACE_REGEX='^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'
 if ${{PERFORMING_UPGRADE_NOT}}; then
-   # Clean up /var directory tree
-   rm -rf /var/opt/microsoft/omsagent 2> /dev/null
-   
-   # Clean up /etc/opt directory tree (created via PostInstall) if dirs are empty
+   # Clean up directory trees: /etc/opt, /var/opt, /opt
    # If multi-workspace scenario
    for ws_id in `ls -1 $ETC_DIR | grep -E $WORKSPACE_REGEX`
     do
         echo "Deleting certs and conf directories for workspace $ws_id if empty..."
-        rmdir /etc/opt/microsoft/omsagent/$ws_id/certs/ 2> /dev/null
-        rmdir /etc/opt/microsoft/omsagent/$ws_id/conf/ 2> /dev/null
-        rm /opt/microsoft/omsagent/bin/omsagent-$ws_id 2> /dev/null		
-        rm /var/opt/microsoft/omsagent/$ws_id/tmp/Changetrackinginventory.xml* 2> /dev/null
```

Comment:
@gauravga Can you comment on this? Before, you were deleting `/var/opt/microsoft/omsagent/$ws_id/tmp/Changetrackinginventory.xml*` and others in the tmp directory. Now I am deleting the entire `/var/opt/microsoft/omsagent/$ws_id/tmp/` directory.